docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -0,0 +1,240 @@
+---
+layout: docs
+title: Command Line Interface
+permalink: /documentation/using-testcafe/command-line-interface/
+---
+# Command Line Interface
+
+TestCafe command line interface is pretty simple.
+It consists of a single command that runs tests with a number of options.
```

Comment:
I feel like we don't need this  two sentences at all. We can remove `Synopsis` header as well, and just insert
```sh
 testcafe [options] <browser-list-comma-separated> <file-or-glob ...>
 ```

right after the `Command Line Interface` header